packages/backend/src/config/parseConfig.ts

```diff
@@ -796,6 +792,56 @@ export const parseConfig = (): LightdashConfig => {
         );
     }
 
+    const rawCopilotConfig = {
+        enabled: process.env.AI_COPILOT_ENABLED === 'true',
+        requiresFeatureFlag:
+            process.env.AI_COPILOT_REQUIRES_FEATURE_FLAG === 'true',
+        embeddingSearchEnabled:
+            process.env.AI_COPILOT_EMBEDDING_SEARCH_ENABLED === 'true',
+        defaultProvider:
+            process.env.AI_DEFAULT_PROVIDER || DEFAULT_DEFAULT_AI_PROVIDER,
+        providers: {
+            azure: process.env.AZURE_AI_API_KEY
+                ? {
+                      endpoint: process.env.AZURE_AI_ENDPOINT,
+                      apiKey: process.env.AZURE_AI_API_KEY,
+                      apiVersion: process.env.AZURE_AI_API_VERSION,
+                      deploymentName: process.env.AZURE_AI_DEPLOYMENT_NAME,
+                  }
+                : undefined,
+            openai: process.env.OPENAI_API_KEY
+                ? {
+                      apiKey: process.env.OPENAI_API_KEY,
+                      modelName:
+                          process.env.OPENAI_MODEL_NAME ||
+                          DEFAULT_OPENAI_MODEL_NAME,
+                  }
+                : undefined,
+            anthropic: process.env.ANTHROPIC_API_KEY
+                ? {
+                      apiKey: process.env.ANTHROPIC_API_KEY,
+                      modelName:
+                          process.env.ANTHROPIC_MODEL_NAME ||
+                          DEFAULT_ANTHROPIC_MODEL_NAME,
+                  }
+                : undefined,
+        },
+    };
+
+    const copilotConfigParse =
+        aiCopilotConfigSchema.safeParse(rawCopilotConfig);
+
+    let copilotConfig: AiCopilotConfigSchemaType;
+    if (!copilotConfigParse.success) {
+        copilotConfig = rawCopilotConfig as AiCopilotConfigSchemaType;
+        Sentry.captureException(copilotConfigParse.error);
+        console.error(
+            `Invalid AI copilot configuration: ${copilotConfigParse.error.message}`,
+        );
+    } else {
+        copilotConfig = copilotConfigParse.data;
+    }
```

Comment:
If there're no problems reported to sentry after deploy, we can just use schema directly